exist-core/src/main/java/org/exist/collections/triggers/CollectionTriggers.java

```diff
@@ -83,72 +86,88 @@ public void configure(DBBroker broker, Txn transaction, Collection col, Map<Stri
 
     @Override
     public void beforeCreateCollection(DBBroker broker, Txn txn, XmldbURI uri) throws TriggerException {
-        for (CollectionTrigger trigger : triggers) {
-            trigger.beforeCreateCollection(broker, txn, uri);
+        if (!triggersSuspended) {
```

Comment:
These `triggersSuspended` checks should not be used here. Such checks are already present at the call site in DBBroker. This kinda breaks the responsibility pattern